app/models/reference.rb

```diff
@@ -28,5 +28,40 @@
 #
 
 class Reference < ApplicationRecord
+  include Auditable
+
   belongs_to :adopter, class_name: 'Adopter'
+
+  after_update :audit_changes
+
```

Comment:
I think the ```Auditable``` concern will work right (and we can get rid of a lot of this code) by just adding something kind of like ```attr_accessor :updated_by_admin_user```.  However it appears that the reference model isn't able to pick up the attribute when it gets passed in at line 102 of the adopters_controller.  Google-Fu is failing me so far.  

@TheDudeWithTheThing @tobymurray

Reply:
I think it still needs to store current user... I found the request_store gem. It acts like Thread.current[], but clears itself after each request.

Reply:
The Adopter model is able to take care of this because the current user is [passed in during the update call](https://github.com/ophrescue/RescueRails/blob/master/app/controllers/adopters_controller.rb#L102).   I haven't been able to figure out how to get that value available during the call of the Audit Concerns, like it is already w/ the adopter model.